components/rss/sources/new-item-from-multiple-feeds/new-item-from-multiple-feeds.ts

```diff
@@ -6,7 +6,7 @@ export default defineSource({
   name: "New item from multiple RSS feeds",
   type: "source",
   description: "Emit new items from multiple RSS feeds",
-  version: "1.0.2",
+  version: "1.0.3",
```

Comment:
I see some code being duplicated between `new-items-from-multiple-feeds` and `new-items-in-feed`. I'd suggest creating a common file that contains all the shared props and a shared `generateMeta` method.